internal/terraform/context_apply2_test.go

```diff
@@ -4012,3 +4013,77 @@ resource "test_object" "x" {
 		t.Fatalf("expected only 'provider oops', but got: %s", msg)
 	}
 }
+
+func TestContext2Apply_excludeListResources(t *testing.T) {
```

Comment:
Isn't this the same test as `TestContext2Plan_excludeListResources`?

Reply:
We probably could remove the plan one, because this apply test also includes a plan command. The plan test however does not execute the apply command.